pkg/platform/aws/aws.go

```diff
@@ -73,6 +73,7 @@ type config struct {
 	HostCIDR                 string            `hcl:"host_cidr,optional"`
 	PodCIDR                  string            `hcl:"pod_cidr,optional"`
 	ServiceCIDR              string            `hcl:"service_cidr,optional"`
+	SetCSIDriverIAMRole      string            `hcl:"set_csi_driver_iam_role,optional"`
```

Comment:
```suggestion
	SetCSIDriverIAMRole      bool            `hcl:"set_csi_driver_iam_role,optional"`
```

Reply:
Oops, didn't catch that I made it a string instead of a bool. Copied anther line and changed the name. Thanks :)